odpf/entropy/v1beta1/service.proto

```diff
@@ -166,3 +172,19 @@ message GetLogRequest {
 message GetLogResponse {
   LogChunk chunk = 1;
 }
+
+message ResourceHistory {
+  string urn = 1;
+  map<string, string> labels = 2;
+  google.protobuf.Timestamp created_at = 3;
```

Comment:
Why no `updated_at`?

Is this for versioning history?

Reply:
Yes this is for resource versioning. The thought is that the versioning should be immutable and append only.